### PR TITLE
Codecov

### PR DESCRIPTION
**Pull request by thomasrockhu**
Hi, Tom from [Codecov](https://codecov.io) here. 

Some of our engineers are huge fans of `tailwindcss`, and I thought it would be useful to add more useful code coverage metrics to this repository. [Codecov](https://codecov.io) is a hosted code coverage provider with awesome features. You can checkout your first reports [here](https://codecov.io/gh/thomasrockhu/tailwindcss).

I made a quick discussion thread [here](https://github.com/tailwindcss/tailwindcss/discussions/1856), but I figured I could just open a PR. Feel free to let me know if you have any questions or comments.

I would love all the feedback you have, and please don't hesitate to reach out.

Cheers,
Tom

Codecov ❤️'s open source and will be free forever for OSS

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
         node-version: [8.x, 10.x, 12.x]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1
```
Comment (thomasrockhu) [inline .github/workflows/nodejs.yml:21]: This is a result of https://github.com/actions/checkout/issues/237

File: package.json [hunk 2]
```diff
@@ -77,6 +77,7 @@
     ]
   },
   "jest": {
+    "collectCoverage":true,
```
Comment (vinayakkulkarni) [inline package.json:80]:
```diff
- "collectCoverage":true,
+ "collectCoverage": true,
```

Comment (thomasrockhu) [reply package.json:80]:
```suggestion
    "collectCoverage": true,
```

Comment (thomasrockhu) [reply package.json]: Whoops, good catch @vinayakkulkarni

**Comment by adamwathan** [comment]
Is it still necessary to use v1 for the checkout action? Don't mind merging but don't want to be on old versions of things if we don't have to be.

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (thomasrockhu) [inline .github/workflows/nodejs.yml:21]:
```suggestion
      - uses: actions/checkout@v2
```

Comment (thomasrockhu) [reply .github/workflows/nodejs.yml]: Looks like this has been patched by GitHub

**Comment by thomasrockhu** [comment]
@adamwathan looks like the issue with `actions/checkout@v2` has been resolved by GitHub.

**Comment by adamwathan** [comment]
Finally merging, thanks!

**Pull request merged**